frontend/assets/stylesheets/components/_payment-b.scss

```diff
@@ -0,0 +1,261 @@
+            width: auto;
+            position: static;
+            left: auto;
+        }
+    }
+
+    .form-section__progress-bar {
+        margin-top: 32px;
+        padding-bottom: 12px;
+        border-bottom: solid guss-colour(news-support-1) 1px;
+    }
+
+    .form-section__name-detail {
+    	margin-top: 7px;
+
+    	.fieldset__heading {
+    		display: none;
+
+    		@include  mq($from: tablet) {
+    			display: table-cell;
+    		}
+    	}
+    }
+
+    .fieldset__headline {
+    	margin-bottom: 0;
+    }
+
+    .form-section__name-detail,
+    .form-section__promotion,
+    .form-section__billing  {
+    	.fieldset {
+    		border-top: 0;
+			}
+    }
+
+    .fieldset__fields {
+        vertical-align: top;
+        padding-top: 2px;
+        padding-right: 5px;
+        padding-left: 5px;
+        padding-bottom: 12px;
+
+        @include mq($from: tablet) {
+            padding-top: 6px;
+        }
+    }
+
+    .form-field__postcode{
+        width: 140px;
+    }
+
+    .form-group{
+        margin-bottom: 24px;
+        border-bottom: 0;
+    }
+
+    .form-group--little-padding {
+        margin-bottom: 12px;
+    }
+
+    .o-inline-list {
+        text-align: right;
+    }
+
+    .o-inline-list__item {
+        font-size: 13px;
+    }
+
+    .pseudo-radio {
+        border-radius: 10px;
+    }
+
+    input[type=radio]:checked ~ .pseudo-radio {
+        border-color: #005689;
+    }
+
+    input[type=radio]:checked ~ .pseudo-radio:before  {
+        background-color: #005689;
+    }
+
+    .u-cf {
+        label {
+            width: 50%;
+
+            @include mq($from: tablet) {
+                width: 49%;
+            }
+
+            float: left;
+        }
+
+        div {
+            width: 50%;
+
+            @include mq($from: tablet) {
+                width: 51%;
+            }
+
+            float: right;
+
+            @include mq($from: tablet) {
+                margin-left: 0;
+            }
+
+            ul > li:nth-child(1) {
+                text-align: left;
+            }
+
+            br {
+                @include mq($from: tablet) {
+                    display:none;
+                }
+            }
+        }
+    }
+
+    .credit-card-bottom-description {
+        @include fs-textSans(2);
+        p {
+            margin-bottom: 0;
+        }
+
+        div {
+            display: block;
+
+            @include mq($from: tablet) {
+                display: none;
+            }
+        }
+    }
+
+    .credit-card-note {
+        display: none;
+
+        @include mq($from: tablet) {
+            display: block;
+        }
+    }
+
+    .promo-code {
+        button {
+            background: transparent;
+            color: #005689;
+            border-color: #005689;
+            background: svg-url('<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30"><path fill="#005689" d="M22.8 14.6L15.2 7l-.7.7 5.5 6.6H6v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"/></svg>') center right no-repeat;
```

Comment:
It appears that we need that attribute so the browser can understand SVGs: https://www.w3.org/TR/SVG/